.devops/code-review-pipelines.yml

```diff
@@ -33,7 +33,7 @@ steps:
       extraProperties: |
         sonar.projectKey=$(SONARCLOUD_PROJECT_KEY)
         sonar.projectName=$(SONARCLOUD_PROJECT_NAME)
-      #  sonar.coverage.exclusions=**/config/*,**/*Mock*,**/model/*
+        sonar.coverage.exclusions=**/config/*,**/*Mock*,**/model/*,**/entity/*
```

Comment:
why exclude `entity` too?

Reply:
for the same reason of `model`. Sonar requires to cover with test every property and every lombok annotation